src/sql/parts/dashboard/dashboardInput.ts

```diff
@@ -90,7 +90,7 @@ export class DashboardInput extends EditorInput {
 			// Only add DB name if this is a non-default, non-master connection
 			name = name + ':' + this.connectionProfile.databaseName;
 		}
-		return name;
+		return this.connectionProfile.connectionName ? this.connectionProfile.connectionName : name;
```

Comment:
This overwrites the logic above on line 91 that modified name to include the database name for database dashboards. Can you move this to line 87 above so that it uses either the server name or the connection name but still appends the database name as needed?

Reply:
Sure thing! Will do now.